admin/src/auth.js

```diff
@@ -111,8 +111,33 @@ class Auth {
             .catch(() => {
                 showError("<h2>Inloggningen misslyckades</h2>Kunde inte kommunicera med servern.");
             });
+
+    }
+
+    update_member_info(tag) {
```

Comment:
`tag` has another meaning in this repo (key fobs), so I'd stick to `member_id` as the argument name